__tests__/lint/enforcements-sort-order-missed-field-error/.redocly.yaml

```diff
@@ -0,0 +1,13 @@
+apiDefinitions:
+  main: ./openapi.yaml
+
+lint:
+  rules:
+    enforcements:
+      - on:
+          - Operation.tags
+        description: Operation tags should be ordered in DESC direction
```

Comment:
I think message here should say something about missing fields in addition to incorrect sort order.

Reply:
The idea behind that is if we set non-existent field it means that the collection of objects is not ordered by that field, we return false. So it is not about missed field. Probably the folder name (which is actually describe's text) is confusing, I'll rename it to `enforcements-sort-order-error-when-missed-field`